cartridges/int_adyen_overlay/cartridge/templates/default/checkout/billing/adyenpaymentfields.isml

```diff
@@ -1,94 +1,43 @@
-				<option class="select-option" value="5">${Resource.msg('month.may', 'forms', null)}</option>
-				<option class="select-option" value="6">${Resource.msg('month.june', 'forms', null)}</option>
-				<option class="select-option" value="7">${Resource.msg('month.july', 'forms', null)}</option>
-				<option class="select-option" value="8">${Resource.msg('month.august', 'forms', null)}</option>
-				<option class="select-option" value="9">${Resource.msg('month.september', 'forms', null)}</option>
-				<option class="select-option" value="10">${Resource.msg('month.october', 'forms', null)}</option>
-				<option class="select-option" value="11">${Resource.msg('month.november', 'forms', null)}</option>
-				<option class="select-option" value="12">${Resource.msg('month.december', 'forms', null)}</option></select>
-		</div>
-		<div class="form-caption"></div>
-	</div>
+<script type="text/javascript">
+	window.onload = function() {
+		var originKey = "${AdyenGetOriginKey.getOriginKeyFromRequest(request.getHttpProtocol(), request.getHttpHost())}";
+		var loadingContext = "${AdyenHelper.getLoadingContext()}";
+		initializeCardComponent(document.getElementById('card'), originKey, loadingContext);
+	};
 
-	<div class="form-row year required" aria-required="true">
-		<label for="creditCard_expiration_year"><span
-			class="required-indicator">• </span><span>${Resource.msg('resource.year', 'forms', null)}</span></label>
-		<div class="field-wrapper">
-			<select class="input-select  required"
-				id="creditCard_expiration_year" minlength="0" maxlength="2147483647"
-				aria-required="true">
-				
-				<isscript>
-					var date = new Date();
-					var currentYear = date.getFullYear();
-					var years = [];
-					
-					for (i = 0; i < 6; i++) {
-						years.push(Number(currentYear + i).toFixed(0));
-					}
-				 </isscript>
-				<isloop iterator="${years}" alias="year">
-					<option class="select-option" value="${year}">${year}</option>
-				</isloop> 				
-			</select>
-		</div>
-		<div class="form-caption"></div>
-	</div>
-</div>
+	function initializeCardComponent(cardNode, originKey, loadingContext) {		
+		var locale = "${request.locale}";
+		var configuration = {
+		        locale: locale, // 'en_US', // Defaults to en_US
+		        translations: {} // Override translations
+		};
 
-<div class="form-row cvn required" aria-required="true">
-	<label for="creditCard_cvn"><span class="required-indicator">•
-	</span><span>${Resource.msg('creditcard.cvnlabel', 'forms', null)}</span></label>
-	<div class="field-wrapper">
-		<input class="input-text  required" type="text" id="creditCard_cvn"
-			value="" aria-required="true">
-	</div>
-	<div class="form-caption"></div>
-</div>
+		var AdyenCheckoutObject = new AdyenCheckout(configuration);
+		
+		window.CardValid = false;
+		window.AdyenCard = AdyenCheckoutObject.create('card', {
+	        // Mandatory fields
+	        originKey: originKey,
```

Comment:
originkey and context should be part of configuration and not recommended to pass in card component.